projects/storefrontlib/src/lib/product/components/product-list/container/product-list.component.ts

```diff
@@ -31,6 +31,7 @@ export class ProductListComponent implements OnChanges, OnInit {
   grid: any;
   model$: Observable<any>;
   searchConfig: SearchConfig = {};
+  pageTitle: string;
```

Comment:
can we rename `pageTitle` to `categoryTitle` or just `title`? `pageTitle` is confusing.